src/main/java/mate/academy/spring/entity/Book.java

```diff
@@ -0,0 +1,56 @@
+package mate.academy.spring.entity;
+
+import static javax.persistence.GenerationType.IDENTITY;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "books")
+public class Book {
+    @Id
+    @GeneratedValue(strategy = IDENTITY)
+    Long id;
```

Comment:
add access modifier for all fields
remember about oop

Reply:
Fixed